build-mono-docker.ps1

```diff
@@ -0,0 +1,7 @@
+trap [Management.Automation.CommandNotFoundException] {
+  Write-Error 'Docker cannot be found. Make sure it is installed and added to the path.'
+  Start-Process -FilePath 'https://docs.docker.com/docker-for-windows/install/'
+  continue;
+}
+
+docker run --rm -it -v ${pwd}:/nunit -w=/nunit mono bash build.sh $args
```

Comment:
Not sure about this, so putting it out for discussion. Should we switch `mono` to `mono:latest` so it is explicit?